torch/_inductor/decomposition.py

```diff
@@ -43,6 +43,7 @@
         aten.dist,
         aten.empty_like,
         aten.flip,
+        aten.frexp,
```

Comment:
We don't need this decomposition for inductor either. Note that the decomposition will be used  as the "meta"-implementation, which does shape and dtype propagation when tracing. So it wasn't a waste of time.